src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/TypeScript/components/MetricCard.tsx

```diff
@@ -54,12 +54,9 @@ const useCardStyles = makeStyles({
     selectedCard: {
         zIndex: 1,
         boxShadow: tokens.shadow8,
-        outline: `2px solid ${tokens.colorNeutralForeground3}`,
```

Comment:
Ah would this make it harder to realize which metric is selected?

Reply:
No, I added a new "radio-button" style to the metric card, so that it is obvious which one is selected without colliding with the focus hint.